simply/scenario.py

```diff
@@ -78,6 +78,9 @@ def save(self, dirpath, data_format):
         # save map_actors
         dirpath.joinpath('map_actors.json').write_text(json.dumps(self.map_actors, indent=2))
 
+        self.power_network.to_image()
+        self.power_network.to_json()
```

Comment:
- network json is already output at line 56 (-> delete to_json)
- to_image writes to the folder from which the script is called, but should be saved within the  scenario folder

Reply:
[6e27665](https://github.com/BESTenergytrade/simply/pull/138/commits/6e2766510982eb8494d69a7e9f3e7815da1cb208)